proxies/live/apiproxy/policies/ServiceCallout.AuthenticateAgainstGPCPFSKeycloak.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Here is the same thing. Keycloack is not the Authorization endpoint for all envs